### ISSUE TITLE
Improve magic str method for Maneuver class

### ISSUE THREAD

**Issue by jorgepiloto** [poliastro/poliastro#873]
💡 **Feature request**

While working with Maneuvers instances, I noticed they do not have a proper `__str__` representation. This class has the attribute `impulses` which already shows information on the maneuver but I thought might be interesting to have something like:

```python
man = Maneuver.hohmann(ss_0, r_f)
print(man)

>>> Hohmann maneuver: ([dt1_value, v1_value], [dt2_value, v2_value], ...)
```

🎯 **Goal and tasks**

This task can be achieved by overriding the [special methods](https://docs.python.org/3/reference/datamodel.html) `__str__` and `__repr__`.

**Comment by iamabhishek0** [comment]
I am working on it :)

**Comment by pandeyroshan** [comment]
Hey Jorge, can you please explain the requirements a little more?

**Comment by priyanshurohilla** [comment]
I think this can be closed now

**Comment by astrojuanlu** [comment]
Yes indeed! Closed in #875. Thanks @priyanshurohilla and @iamabhishek0!
